docs/UserGuide.adoc

```diff
@@ -286,36 +289,52 @@ For example, if you enter the wrong name and want to undo the action, type `/und
 
 .Form layout showing that the last entry is removed after you typed in `/undo`
 image::undoCommand.png[width="790"]
+// end::add[]
 
+// tag::list[]
 === Listing persons : `list`
+This list command can show you the list of persons in ORGANice.
 
-Shows a list of all doctors, donors, or patients in ORGANice, depending on the type of person specified. Shows all
-persons in ORGANice if no type is specified.
+==== Listing all persons
+Simply type `list` in the Command Box and ORGANice will show you all patients, doctors and donors present in the system.
 
 Format: `list`
 
-Optional parameters: Type parameter - `t/TYPE`
+==== Listing persons of a specified type
+If you wish to see a certain type of person only, you can provide the type parameter in the list command.
 
-Examples:
+Optional parameter: Type - `t/TYPE`
 
-* `list` +
-Lists all patients, doctors and donors in the records.
-* `list t/patient` +
-Lists all patients only.
+For instance, if you only want to see all doctors, you can enter the following:
+`list t/doctor`. Likewise, you can change the type parameter in the same manner to see only patients or donors.
+// end::list[]
 
-=== Editing a person : `edit`
+// tag::edit[]
+=== Editing a person's attribute : `edit`
+If you want to make any changes to a person's attribute, you can edit the information using an edit command.
+If there are multiple attributes to change, you can do so in the same edit command.
 
-Edits the information of the person with the specified NRIC to the attributes specified.
-It is up to the user to specify which attributes should be updated. +
+The format is such that after the `edit` keyword, provide the person's NRIC, followed by the attribute(s) that you want to change in the edit command.
 
-Format: `edit **NRIC**`
+Format: `edit **NRIC**` followed by attributes to update
+
+For instance, if you want to change a patient's phone number to 91234567, you can provide his or her NRIC followed by the new phone number
+as follow: `edit S8732457G p/91234567`.
+
+**You can only make changes to attributes that the person has.**
+For example, you cannot change a doctor's age because a doctor does not have that attribute in ORGANice.
+
+**In addition, there are some attributes that cannot be edited.** These include:
+
+* NRIC
++
+* Type
++
+* Organ expiry date
```

Comment:
Can I ask, why is it that organ expiry date cannot be changed?

Reply:
Will clarify again with Simnee.